src/components/CCard.vue

```diff
@@ -108,17 +113,48 @@ withDefaults(
   {
     title: undefined,
     titlePosition: "left",
+    titleSize: "normal",
     subtitle: undefined,
     width: "100%",
     maxWidth: "850px",
   },
 );
+
+/**
+ * Computes the classes for the title component.
+ */
+const titleClass = computed<string>(() => {
+  let classes = "";
+
+  classes += "text-" + props.titlePosition;
+
+  if (props.titleSize && props.titleSize !== "normal")
+    classes += " " + props.titleSize;
+
+  return classes;
+});
 </script>
 
 <style scoped lang="scss">
-.c-card .c-card-sections > * {
-  &:not(:last-child) {
-    margin-bottom: 12px;
+@import "vuetify/lib/styles/settings/variables";
+
+.c-card {
+  .c-card-sections > * {
+    &:not(:last-child) {
+      margin-bottom: 12px;
+    }
+  }
+
+  .v-card-title.large {
+    line-height: normal;
+
+    @media (max-width: map-get($grid-breakpoints, "md")) {
+      font-size: 26px !important;
+    }
+
+    @media (min-width: map-get($grid-breakpoints, "md")) {
+      font-size: 30px !important;
+    }
```

Comment:
what happens exactly at the border?

Reply:
both should be including the border, so any of these two will win